docs/add-ons/theme-addon.mdx

```diff
@@ -168,50 +175,25 @@ final yellowTheme = AppThemeData(
 );
 ```
 
-<Tabs
-  groupId="approach"
-  values={[
-    { label: 'Generator', value: 'generator' },
-    { label: 'Manual', value: 'manual' },
-  ]}
->
-  <TabItem value="generator">
-    To use the `CustomThemeAddon` with the [widgetbook_generator package](https://pub.dev/packages/widgetbook_generator) annotate a function that returns your `AppThemeData` with the `@WidgetbookTheme` annotation.
-
-    ```dart title="Example: @WidgetbookTheme for custom theme"
-    @WidgetbookTheme(name: 'Blue', isDefault: true)
-    AppThemeData getTheme() => blueTheme;
-    ```
-
-    <Info>
-        You can use the `@WidgetbookTheme` annotation on multiple theme instances.
-    </Info>
-  </TabItem>
-  <TabItem value="manual">
-    The `CustomThemeAddon` requires a `CustomThemeSetting` that has a generic parameter and two properties.
-    The `themes` property defines which themes are available while the `activeTheme` property defines which theme is active and selected.
-
-    ```dart title="Example: CustomThemeAddon"
-    final themeAddon = CustomThemeAddon<AppThemeData>(
-      setting: CustomThemeSetting(
-        themes: [
-          WidgetbookTheme(
-            name: 'Blue',
-            data: blueTheme,
-          ),
-          WidgetbookTheme(
-            name: 'Yellow',
-            data: yellow,
-          ),
-        ],
-        activeTheme: WidgetbookTheme(
-            name: 'Blue',
-            data: blueTheme,
-        ),
-      ),
-    );
-    ```
-  </TabItem>
-</Tabs>
-
-<Warning>For the theme to be properly injected into Widgetbook's Widget tree, make sure to implement a custom [`appBuilder` method](/getting-started/configuration#customize-how-use-cases-are-previewed).</Warning>
\ No newline at end of file
+The `CustomThemeAddon` requires two properties. The `themes` property defines
+which themes are available while the `initialTheme` property defines which theme
+is active and selected.
+
+```dart title="Example: CustomThemeAddon"
+final themeAddon = CustomThemeAddon<AppThemeData>(
+  themes: [
+    WidgetbookTheme(
+      name: 'Blue',
+      data: blueTheme,
+    ),
+    WidgetbookTheme(
+      name: 'Yellow',
+      data: yellowTheme,
+    ),
+  ],
+  initialTheme: WidgetbookTheme(
+      name: 'Blue',
+      data: blueTheme,
+  ),
+);
+```
```

Comment:
This also requires a builder function that makes sure the inherited widget is injected.